src/components/kyb/ComplianceStep.tsx

```
@@ -0,0 +1,273 @@
+    
+    return `${baseClasses} border border-gray-600`;
+  };
+
+  // Expose validation function to parent component
+  useEffect(() => {
+    const allRequiredChecked = data.accurateInformation &&
+                              data.notIllegalActivities &&
+                              data.additionalInfoConsent &&
+                              data.privacyPolicyConsent;
+    // Only update if the validation status has changed
+    if ((data as any)._complianceStepValid !== allRequiredChecked) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)